src/pages/LoginAndRegister/RegisterSocial.vue

```diff
@@ -221,7 +259,27 @@ export default {
 				this.$kvTrackEvent('Register', 'error-register-social-form-invalid-input');
 			}
 		},
+		fetchStrategicPartnerLoginInfoByPageId() {
+			this.apollo.query({
+				query: StrategicPartnerLoginInfoByPageId,
+				variables: { pageId: this.partnerContentId }
+			}).then(response => {
+				const { logo } = response.data.strategicPartnerLoginInfoByPageId;
```

Comment:
Is `logo` guaranteed to be defined? Do we need optional chaining here and below where `logo` is used?

Reply:
Optional chaining added.  Done